pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -184,6 +207,18 @@ func (s *socketsFirst) sortAvailableSockets() []int {
 	return sockets
 }
 
+// If sockets  higher in the memory hierarchy than NUMA nodes, then uncorecache
+// sit directly below NUMA Nodes in the memory hierchy
+func (s *socketsFirst) sortAvailableUnCoreCaches() []int {
+	var result []int
+	for _, uncore := range s.acc.sortAvailableNUMANodes() {
```

Comment:
why `uncore`?
In addition, this logic seems identical to the `numaFirst` case, is that intentional?

Reply:
Yes, this is intentional, we have a couple methods here defined that follow the same logic

Reply:
thanks @wongchar, will review again more carefully.